src/emailalert/email_function.py

```diff
@@ -10,16 +10,18 @@ def lambda_handler(event, context):
     SUBJECT = "Failure alert for RDS Restore Pipeline"
     result = {}
     if 'status' in event:
-        result[constants.ERROR] = event['taskname'] + 'Error'
-        result[constants.CAUSE] = event['status']
+        result['failed step'] = event['taskname'] + 'Error'
+        result['cause of failure'] = event['status']
     elif 'Error' in event:
-        result[constants.ERROR] = event['Error']
-        result[constants.CAUSE] = event['Cause']
+        result['database id'] = event['DatabaseID']
```

Comment:
the event will contain `identifier` for database id and snapshot id is inferred from the database id. sharing a code reference https://github.com/intuit/Trapheus/blob/master/src/snapshot/snapshot_function.py#L11 

lets use the same here